content/starter-configs/tutorials/wordpress.md

```diff
@@ -48,62 +50,106 @@ define('DB_PASSWORD', 'tugboat');
 define('DB_HOST', 'mysql');
 ```
 
-## Configure Tugboat
+_**🐙 Note:** Make sure your `wp-config.php` is formatted as close to the sample file as possible. The Wordpress CLI
+gets angry when it's not and will release the Kraken on your Tugboat builds._
```

Comment:
😆 👏 

Note that emoji may not display on some OSes. I believe we would need to add an emoji font or something to our website for these to work consistently across all OSes.